src/content/resources/developer-resources/typescript/typescript-getting-started.mdx

```diff
@@ -73,18 +65,20 @@ npm install --save-dev typescript
 **Initialize a TypeScript project with:**
 
 ```bash
-tsc --init  
+tsc --init
 ```
+
 This creates a `tsconfig.json` file to configure TypeScript options.
 
 <br />
-Optional: If you’re using Node.js directly, 
-you can install ts-node to run TypeScript files without precompiling:
+Optional: If you’re using Node.js directly, you can install ts-node to run
+TypeScript files without precompiling:
 
 ```bash
 npm install -g ts-node
 ts-node yourfile_name.ts
 ```
+
 </TextContainer>
 
 <TextContainer background="light">
```

Comment:
Let's remove the background here. We don't need to add background color to sections, except if the section is not related to the topic but needs to be there, such as "Appreciation".


```suggestion
<TextContainer>
```